python/cuml/common/sparsefuncs.py

```diff
@@ -73,6 +73,23 @@ def _map_l2_norm_kernel(dtype):
     return cuda_kernel_factory(map_kernel_str, dtype, "map_l2_norm_kernel")
 
 
+def _map_mul_sparse_array(dtype):
+    """Creates cupy RawKernel for multiplying sparse array with a dense ar """
+
+    map_kernel_str = r'''
+    ({0} *data, {1} *indices, {2} *indptr, {3} *dense_ar, int data_size) {
+
+      int tid = blockDim.x * blockIdx.x + threadIdx.x;
+
+      if(tid >= data_size) return;
+
+      int index = indices[tid];
```

Comment:
If indices is of type `{1}`, you also want to use that type for `index`

Reply:
Resolved as we got rid of the kernel with  https://github.com/rapidsai/cuml/commit/1efcbde48a7077c2522e70587d96fe68d6fee11c  .